cmd/main.go

```diff
@@ -152,10 +167,18 @@ func (c Configuration) fileBelongsToExclusionList(file string) bool {
 	return false
 }
 
-func (c Configuration) genericConsumer(fs afero.Fs) []GenericFile {
+// Gets list of files to be monitored from all files/dirs listed in the config
+func (c Configuration) getListOfFiles(fs afero.Fs, consumerType string) []FileInfo {
 	logger := c.logger()
-	var genericFiles []GenericFile
-	for _, fullPath := range c.Consumers.Generic {
+	var filesToMonitor []FileInfo
+	var pathList []string
+	switch consumerType {
+	case "sudoers":
```

Comment:
Try to get rid of `consumerType`, just pass `the c.Consumers.Sudoers` or `c.Consumers.Generic` directly

Reply:
Updated code to pass the list instead.